runtime/kusama/src/lib.rs

```diff
@@ -556,12 +570,12 @@ impl pallet_staking::Config for Runtime {
 }
 
 parameter_types! {
-	pub const LaunchPeriod: BlockNumber = 7 * DAYS;
-	pub const VotingPeriod: BlockNumber = 7 * DAYS;
-	pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
+	pub LaunchPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1, "KSM_LAUNCH_PERIOD");
```

Comment:
```suggestion
	pub LaunchPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1 * DAYS, "KSM_LAUNCH_PERIOD");
```

Reply:
The 1 is intentional. When testing (at least in my tests) you don't want a long queue of proposals. You want to vote as quickly as possible on the proposal you put up.